common/elasticsearch/esql/README.md

```diff
@@ -0,0 +1,162 @@
+# ESQL: Translate SQL to Elasticsearch DSL
+Use SQL to query Elasticsearch. ES V6 compatible.
+
+## Supported features
+- [x] =, !=, <, >, <=, >=, <>, ()
+- [x] comparison with arithmetics (e.g. colA + 1 < colB * 2)
+- [x] arithmetic operators: +, -, *, /, %, >>, <<, ()
+- [x] AND, OR, NOT
```

Comment:
nit, the format is italic, good to make them consistent

Reply:
It's not italic on my side...